prodsys/simulation/control.py

```diff
@@ -433,14 +444,30 @@ def start_process(self) -> Generator:
                         ],
                     )
                 logger.debug({"ID": "controller", "sim_time": self.env.now, "resource": self.resource.data.ID, "event": f"Starting picking up {product.product_data.ID} for transport"})
-                yield self.env.process(
-                    self.run_process(transport_state, product, target=origin, empty_transport=True)
-                )
+
+                if isinstance(process, LinkTransportProcess) or isinstance(process, RequiredCapabilityProcess):
```

Comment:
This looks a little bit full -> move to extra function or make logging more efficient.